utah-design-system-website/src/react/components/websiteContent/Search.jsx

```diff
@@ -19,9 +20,56 @@ export function Search() {
     if (param) { setQuery(param); }
   };
 
+  const checkSkipLink = () => {
+    const link = document.getElementById('skip-link');
+    if (link) {
+      setSkipLink(link);
+    } else {
+      // Create observer to watch for new nodes
+      const obs = new MutationObserver(() => {
+        const link = document.getElementById('skip-link');
+        if (link) {
+          setSkipLink(link);
+        }
+      });
+      // @ts-ignore
+      if (!observer) setObserver(obs);
+    }
+  };
+
+  const setSkipLink = (/** @type {HTMLElement} */ link) => {
+    // Update skip link
+    link.onclick = (e) => {
```

Comment:
what if `link` already has an `onclick` event? then this would clobber it and cause a bug. Maybe that's not plausible since we own the link element? Some options include throwing an error if onclick is already set, or chaining the existing onclick to the new onclick so it isn't forgotten.